src/main/java/seedu/address/ui/WarningWindow.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.ui;
+
+import javax.swing.JFrame;
+import javax.swing.JOptionPane;
+
+/**
+ * This class create a warning window. Future upgrade include adding an extension to
+ * UIPart to utilize fxml and standardise to the rest of the class
+ * under ui package.
+ */
+public class WarningWindow {
+    private JFrame frame = new JFrame("");
+    private String message = "";
+
+    public WarningWindow() {
+    }
+
+    public WarningWindow(String presetMessage) {
+        message = presetMessage;
+    }
+
+    /**
+     * Creates the pop up warning window with the indicated frame and message.
```

Comment:
Consider add one more empty line here to separate the first sentence from @return sentence.